### PR TITLE
Add type for fingerpose

### PR DESCRIPTION
**Pull request by EthanL06**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/fingerpose/index.d.ts
1. types/fingerpose/fingerpose-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 61ff715366c26307c867038a3fe735e1427bfb51
</p>

File: types/fingerpose/tslint.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "extends": "@definitelytyped/dtslint/dt.json" }
```
Comment (jakebailey) [inline types/fingerpose/tslint.json:1]: You should remove this file; it is no longer required.

**Review by jakebailey — CHANGES_REQUESTED** [review]

File: types/fingerpose/tslint.json (hunk 1 above)
Comment (EthanL06) [reply types/fingerpose/tslint.json]: Fixed.

File: types/fingerpose/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
+export enum Finger {
```
Comment (jakebailey) [inline types/fingerpose/index.d.ts:1]: These shouldn't be enums unless the original code was somehow generated by TypeScript or has the same runtime format.

Upstream, they all consist of objects with values, mappings, functions, etc: https://github.com/andypotato/fingerpose/blob/master/src/FingerDescription.js#L1

They wouldn't be compatible with TypeScript's interpretation of a runtime enum.

Comment (EthanL06) [reply types/fingerpose/index.d.ts]: Fixed. I changed it from enum to objects

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@EthanL06 thanks!

**Comment by EthanL06** [comment]
Ready to merge

**Pull request merged**